lib/src/file-input/FileInput.tsx

```diff
@@ -179,7 +179,6 @@ const DxcFileInput = React.forwardRef<RefType, FileInputPropsType>(
                 accept={accept}
                 multiple={multiple}
                 onChange={selectFiles}
-                name={name}
                 disabled={disabled}
                 readOnly
                 aria-hidden="true"
```

Comment:
One comment here. I recently discovered that aria-hidden is not necessary when an element is hidden using `display: none;`

![image](https://user-images.githubusercontent.com/44321109/222372688-e94d2969-110e-4e80-b05d-9485336fe55a.png)


https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden

Can you please remove it from this hidden input?

Reply:
Totally agree!
Done.